app/src/main/java/com/rahobbs/todo/helpers/TodoLab.java

```diff
@@ -0,0 +1,119 @@
+package com.rahobbs.todo.helpers;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.rahobbs.todo.database.TodoBaseHelper;
+import com.rahobbs.todo.database.TodoCursorWrapper;
+import com.rahobbs.todo.database.TodoSchema.TodoTable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/2/16.
+ * Creates an array list of the items to display
```

Comment:
be more specific here... it's a wrapper around the database to access the information.